### PR TITLE
Fix multiple authorization checks

### PR DESCRIPTION
**Pull request by whyDontI**
Fixes https://github.com/Real-Dev-Squad/website-status/issues/305

- [x] Removed error toast which wasn't necessary for Non-Admin users
- [x] Removed useEffect authorization on card component, used `shouldEdit` prop passed from index page
- [x] Lint fixes on the edited pages

### CODE REVIEW

File: src/components/tasks/card/index.tsx [hunk 1]
```diff
@@ -41,8 +41,8 @@ const Card: FC<Props> = ({
       setShowEditButton(true)
     }
   }, [keyLongPressed]);
-  
-  const context = useAppContext() ;
```
Comment (Maheima) [inline src/components/tasks/card/index.tsx:45]: Can you please fix these unnecessary changes that show up here?

**Review by Maheima — CHANGES_REQUESTED** [review]
There are a lot of unnecessary addition of space-related changes in this PR. Please fix those so, it becomes cleaner and not distracting

File: src/pages/index.tsx [hunk 2]
```diff
@@ -107,26 +107,23 @@ const Index: FC = () => {
...
-        const { data } = await requestPromise;
-        const userRoles = {
-          adminUser: data.roles?.admin,
-          superUser: data.roles?.super_user,
-        };
-        const { adminUser, superUser } = userRoles;
-        setIsUserAuthorized(!!adminUser || !!superUser);
-      } catch (err: any) {
-        toast(ERROR, err.message);
-      }
```
Comment (ankushdharkar) [inline src/pages/index.tsx:128]: Should we not convert the toast to maybe a console error at least?

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

**Comment by whyDontI** [comment]
Same issue is being solved in [this](https://github.com/Real-Dev-Squad/website-status/pull/306) PR, should we close this one?

**Comment by akshay1502** [comment]
Closing this PR, https://github.com/Real-Dev-Squad/website-status/pull/306 resolved the issue.

**Pull request closed without merging**